sass/directives/_tables.scss

```diff
@@ -40,64 +40,19 @@
 
 
 
-@mixin table--row-column-headers($table-highlight-z-index: 1, $table-caption-z-index: 2) {
-  border-collapse: separate;
-  border-spacing: $table-border-spacing 0;
-  table-layout: fixed;
-  width: 100%;
+@mixin table--row-column-headers($table-highlight-z-index: 1, $table-caption-z-index: 2, $table-col-width: 25%) {
```

Comment:
The content format mixin needs the `$table-col-width: 25%` param (does it not break without it?)

I'd also want to move the highlight stuff and the z-index params that go with it to the content format mixin, b/c it's a behavior that's specific to that particular table format, but I see you're leaving the highlight on the product table and only removing it from the ecom table so this is fine for now